lib/pages/surveydetail/survey_detail_page.dart

```diff
@@ -55,8 +66,47 @@ class _SurveyDetailPageState extends ConsumerState<SurveyDetailPage> {
     return Scaffold(
       resizeToAvoidBottomInset: false,
       body: survey != null
-          ? StartSurvey(survey: survey)
+          ? Stack(
+              children: [
+                Container(
+                  decoration: BoxDecoration(
+                    image: DecorationImage(
+                      image: Image.network(survey.coverImageUrl).image,
+                      fit: BoxFit.cover,
+                    ),
+                  ),
+                ),
+                DimmedBackground(
+                  colors: [
+                    Colors.black.withOpacity(0.2),
+                    Colors.black.withOpacity(0.7),
+                  ],
+                  stops: const [0.0, 1.0],
+                ),
+                _buildSurveyQuestionPager(survey)
+              ],
+            )
```

Comment:
What do you think about creating a reusable widget for this one, we duplicated it somewhere right?
```dart
DimedBody(coverImage: Image.network(survey.coverImageUrl).image, body: _buildSurveyQuestionPager(survey))
```

Reply:
@markgravity Fixed by https://github.com/luongvo/flutter-survey/pull/64/commits/1720d9d922debbb1f476d091419a67e16711014b